kibana/tests/kibana_test.py

```diff
@@ -41,6 +41,9 @@ def test_defaults():
 
     assert r['deployment'][name]['spec']['strategy']['type'] == 'Recreate'
 
+    # Make sure that the default 'annotation' dictionary is empty
+    assert not r['service'][name]['metadata']['annotations']
```

Comment:
This one is failing in CI with:

```
08:23:59 >       assert not r['service'][name]['metadata']['annotations']
08:23:59 E       KeyError: 'annotations'
```

I think you want:

```
assert 'annotations' not in r['service'][name]['metadata']
```

Reply:
Alright, let me fix this real quick